easybuild/framework/easyblock.py

```diff
@@ -1444,8 +1444,11 @@ def extensions_step(self, fetch=False):
         self.clean_up_fake_module(fake_mod_data)
 
     def package_step(self):
-        """Package software (e.g. into an RPM)."""
-        pass
+        """Prepare package software (e.g. into an RPM) with fpm."""
+
```

Comment:
Perhaps, there is one sane direction to propose about this; let's deliver built rpms on a per dict basis:

`output_subdirectory = ['GPL': 'public/', 'MIT': 'public/', 'JavaCP': 'private/', '': 'private']`

People then may choose, if they fence it via permissions or some other mechanism.

If the idea passes, this could well be part of easybuild configuration, for extensibility (alike moduleclasses).